package.json

```diff
@@ -27,8 +27,9 @@
   },
   "devDependencies": {
     "codeclimate-test-reporter": "0.0.4",
+    "gh-release": "^1.1.7",
     "istanbul": "^0.3.13",
-    "semistandard": "^4.0.3",
+    "semistandard": "^4.1.0",
```

Comment:
@flet is there an internet place where it's already been discussed / elaborated on as to  why `*` is best for semistandard/standard? would like to be able to point someone to it rather than just say "'cause I said so"